docs/field-values.asciidoc

```diff
@@ -361,6 +400,16 @@ The deletion event type is used for the subset of events within a category that
 
 
 
+[float]
+[[ecs-event-type-denied]]
+==== denied
+
+The denied event type is used for the subset of events within a category that indicate that something was denied. Common examples include `event.category:”network” AND event.type:denied` (to indicate a network firewall event for which the firewall disposition was to deny the connection to complete) and `event.category:intrusion_detection AND event.type:denied` (to indicate a network intrusion prevention system event for which the IPS disposition was to deny the connection to complete). You can further distinguish denied operations using the ECS `event.action` field, populating with values of your choosing, such as "blocked", "dropped", or "quarantined".
```

Comment:
The help keep people sane 😄 replace the usage of right (`”`) quotes with a plain old quote (`"`). Or just remote that set of quotes since the other term in the same query doesn't use quotes.

Reply:
OK, deleted them.